examples/disaster-recovery/failover.sh

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_sql_proxy_secret_manager_failover]
```

Comment:
This region tag will need to change so it won't conflict with the v1 region tag, right?

Reply:
Actually I believe we want to have it be replaced with the v2 example by default.

Reply:
Yeah, that's a better idea.